crates/mitex/tests/cvt/arg_match.rs

```diff
@@ -0,0 +1,161 @@
+use super::prelude::*;
+
+#[test]
+fn curly_group() {
+    assert_debug_snapshot!(convert_math(r#"a \textbf{strong} text"#), @r###"
+    Ok(
```

Comment:
We can `unwrap` or `unwrap_err` the `Result` to reduce an indent here.

Reply:
Then we can also use assert_snapshot instead of assert_debug_snapshot to avoid excessive string escapes, as the result of `convert_math` is a string type.

Reply:
Thanks, fixed it.